Sming/Core/Network/Http/HttpRequest.h

```diff
@@ -119,6 +119,11 @@ class HttpRequest
 		return this;
 	}
 
+	String methodStr()
```

Comment:
Please, change the name of the method to `String getMethod()`. And add a doc block.

Reply:
I don't think `getMethod()`  is appropriate as it implies we're getting the enumerated type. I considered `getMethodString()` and `methodString()` but we already have `BasicHttpHeaders::methodStr()`. Maybe a better way to deal with this is to define `HttpMethod` as a strong enum (shadowing `http_method`) and add a regular `toString` function for it?

Reply:
> ... Maybe a better way to deal with this is to define HttpMethod as a strong enum (shadowing http_method) and add a regular toString function for it?

Yes, let's do it this way. Will C++ at some point complain that there is a function `String toString(X x)` that is so heavily overloaded?

Reply:
Nope. Each and every instance of `toString` has a different (mangled) name **provided** the types are strong.